wordpress_rock/rockcraft.yaml

```
@@ -0,0 +1,186 @@
+      # Required to source $CRAFT_OVERLAY/etc/apache2/envvars
+      - APACHE_CONFDIR: /etc/apache2
+      - IMAGE_RUN_USER: _daemon_
+      - IMAGE_RUN_GROUP: _daemon_
+      - IMAGE_RUN_USER_ID: 584792
+      - IMAGE_RUN_GROUP_ID: 584792
+    override-pull: |
+      # Persists files here to use them in overlay
+      craftctl default
+      rm -rfv /tmp/craft_apache_conf_files/*
+      mkdir -p /tmp/craft_apache_conf_files
+      cp -R . /tmp/craft_apache_conf_files
+    overlay-script: |
+      craftctl default
+      sed -ri 's/^export ([^=]+)=(.*)$/: ${\1:=\2}\nexport \1/' $CRAFT_OVERLAY/etc/apache2/envvars
+      sed -ri 's/\{APACHE_RUN_(USER|GROUP):=.+\}/\{APACHE_RUN_\1:=_daemon_\}/' $CRAFT_OVERLAY/etc/apache2/envvars
+      . $CRAFT_OVERLAY/etc/apache2/envvars
+      for dir in "$CRAFT_OVERLAY$APACHE_LOCK_DIR" "$CRAFT_OVERLAY$APACHE_RUN_DIR" "$CRAFT_OVERLAY$APACHE_LOG_DIR";
+        do
+          rm -rvf "$dir";
+          mkdir -p "$dir";
+          chown "$IMAGE_RUN_USER_ID:$IMAGE_RUN_GROUP_ID" "$dir";
+          chmod u=rwx,g=rx,o=rx "$dir";
+      done
+      chown -R --no-dereference "$IMAGE_RUN_USER_ID:$IMAGE_RUN_GROUP_ID" "$CRAFT_OVERLAY$APACHE_LOG_DIR"
+      ln -sfT ../../../dev/stdout "$CRAFT_OVERLAY$APACHE_LOG_DIR/other_vhosts_access.log"
+      mv /tmp/craft_apache_conf_files/docker-php.conf $CRAFT_OVERLAY/etc/apache2/conf-available/docker-php.conf
+      mv /tmp/craft_apache_conf_files/docker-php-swift-proxy.conf $CRAFT_OVERLAY/etc/apache2/conf-available/docker-php-swift-proxy.conf
+      mv /tmp/craft_apache_conf_files/apache2.conf $CRAFT_OVERLAY/etc/apache2/apache2.conf
+      mv /tmp/craft_apache_conf_files/000-default.conf $CRAFT_OVERLAY/etc/apache2/sites-available/000-default.conf
+
+      # Enable apache2 modules
+      chroot $CRAFT_OVERLAY /bin/sh -x <<'EOF'
+      a2enconf docker-php
+      a2enmod headers
+      a2enmod mpm_prefork
+      a2enmod proxy
+      a2enmod proxy_http
+      a2enmod rewrite
+      a2enmod ssl
+      EOF
+  wordpress:
+    after:
+      - apache2
+    plugin: nil
+    build-environment:
+      - WP_PLUGINS: |
+          404page \
+          all-in-one-event-calendar \
+          coschedule-by-todaymade \
+          elementor \
+          essential-addons-for-elementor-lite \
+          favicon-by-realfavicongenerator \
+          feedwordpress \
+          genesis-columns-advanced \
+          line-break-shortcode \
+          no-category-base-wpml \
+          post-grid \
+          powerpress \
+          redirection \
+          relative-image-urls \
+          rel-publisher \
+          safe-svg \
+          show-current-template \
+          simple-301-redirects \
+          simple-custom-css \
+          so-widgets-bundle \
+          svg-support \
+          syntaxhighlighter \
+          wordpress-importer \
+          wp-font-awesome \
+          wp-lightbox-2 \
+          wp-markdown \
+          wp-mastodon-share \
+          wp-polls \
+          wp-statistics
```

```suggestion
      - WP_PLUGINS: >-
          404page
          all-in-one-event-calendar
          coschedule-by-todaymade
          elementor
          essential-addons-for-elementor-lite
          favicon-by-realfavicongenerator
          feedwordpress
          genesis-columns-advanced
          line-break-shortcode
          no-category-base-wpml
          post-grid
          powerpress
          redirection
          relative-image-urls
          rel-publisher
          safe-svg
          show-current-template
          simple-301-redirects
          simple-custom-css
          so-widgets-bundle
          svg-support
          syntaxhighlighter
          wordpress-importer
          wp-font-awesome
          wp-lightbox-2
          wp-markdown
          wp-mastodon-share
          wp-polls
          wp-statistics
```

We could consider using the folded block scalar `>` in YAML, as the escape character `\` may not work properly within block scalars.

---

Thanks! I updated it to a folded block scalar syntax